data/_ui/interfaces.txt

```diff
@@ -800,6 +800,17 @@ interface "planet"
 		align right
 		pad 10 0
 
+	visible if "!has ship"
+	sprite "ui/planet dialog button"
+		center 280 270
+	button "" "Depart"
+		center 280 270
+		dimensions 140 40
+		size 18
+		align right
+		pad 10 0
+
+
```

Comment:
Don't do this. Instead, change the `visible if` on the entry above this one to `active if`.
Also, a button that doesn't trigger a key is just a [label](https://github.com/endless-sky/endless-sky/wiki/CreatingInterfaces#labels-strings-and-buttons).

Reply:
Ah, thanks.  I tried various things, but I didn't run across those.

Reply:
You neglected to mention that changing the visible if to active if meant I needed to close the previous visible if with a visible. ;)